### DISCUSSION TITLE
Some other program is currently using the package manager

### DISCUSSION THREAD

**Discussion by tgwaste** [cockpit-project/cockpit#16196] [Q&A] [answered]
```
# ls -la /var/lib/dpkg/lock
ls: cannot access '/var/lib/dpkg/lock': No such file or directory
```

How to fix this?
(without rebooting)

Version
248-1~ubuntu20.04.1

**Comment by martinpitt** [comment, marked as answer]
This message comes from PackageKit. Possibly you were running Gnome Software, or perhaps manually ran `pkcon refresh` or something similar?
